### PR TITLE
Telemetry channel should not be prefixed

### PR DESCRIPTION
**Pull request by patrick-gu**
## Description
Jack said this was what to be changed

## Developer Testing

Here's what I did to test my changes:

- I didn't but I assume this is right?

## Reviewer Testing

Here's what you should do to quickly validate my changes:

- Please check that this makes sense

- - -
This change is [Reviewable](https://reviewable.io/reviews/waterloo-rocketry/omnibus/279)

### CODE REVIEW

File: sources/parsley/main.py [hunk 1]
```diff
@@ -121,10 +121,7 @@ def main():
     else:
         parser = parsley.parse_usb_debug
 
-    if args.format == "telemetry":
-        channel = "telemetry/" + RECEIVE_CHANNEL
-    else:
-        channel = RECEIVE_CHANNEL
+    channel = RECEIVE_CHANNEL
```
Comment (BluCodeGH) [inline sources/parsley/main.py:124]: Pls just delete the channel variable and update where it's used directly

Comment (patrick-gu) [reply sources/parsley/main.py]: Done. Functionally though, is it correct?

**Review by Darth-Raazi — APPROVED** [review]
:lgtm: but this needs to be tested. @BluCodeGH can you suggest how to test?

> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/omnibus/279#-:-O2h3Juv3O1PAwCIQnVH:b-33n2di)* status: all files reviewed, 1 unresolved discussion (waiting on @BigLiban, @BluCodeGH, @Lucifersan, and @saads312)

**Review by BluCodeGH — APPROVED** [review]
LGTM as well, we can test during the benchtop testing today.

**Pull request merged**